plugins/native/module/vtkQuakeMDLImporter.h

```diff
@@ -0,0 +1,133 @@
+/**
+ * @class   vtkQuakeMDLImporter
+ * @brief   VTK Importer for Quake 1 models in binary .mdl file format
+ */
+
+#ifndef vtkQuakeMDLImporter_h
+#define vtkQuakeMDLImporter_h
+#include <vtkImporter.h>
+
+class vtkQuakeMDLImporter : public vtkImporter
+{
+public:
+  static vtkQuakeMDLImporter* New();
+  vtkTypeMacro(vtkQuakeMDLImporter, vtkImporter);
+
+  void PrintSelf(ostream& os, vtkIndent indent) override;
+
+  ///@{
+  /**
+   * Set/Get the file name.
+   */
+  vtkSetMacro(FileName, std::string);
+  vtkGetMacro(FileName, std::string);
```

Comment:
you can remove this method

Reply:
Should I implement it outside the header ?

Reply:
I mean you can remove the vtkGetMacro(Filename, std::string), it is not used as far as I can tell.